NEWS

```diff
@@ -1,3 +1,9 @@
+* Version 2.7 released 2020-02-??
```

Comment:
put this as "unreleased"? until it's actually out?

Reply:
Sure